File: grades.py
import sys

name = raw_input ("Please enter your name: ")

assignment = raw_input ("Assignment: ")

grades = ["A", "B", "C", "D", "F", "INVALID GRADE"]

gradeStr = raw_input ("Thank you. Please enter the grade you wish to convert into a letter: ")

grade = int(gradeStr)

if grade <= 100 and grade > 89:
    print(name + ", your letter grade for " + assignment + " is " + grades[0])
elif grade <= 89 and grade > 79:
    print(name + ", your letter grade for " + assignment + " is " + grades[1])
elif grade <= 79 and grade > 69:
    print(name + ", your letter grade for " + assignment + " is " + grades[2])
elif grade <= 69 and grade > 59:
    print(name + ", your letter grade for " + assignment + " is " + grades[3])
elif grade < 60 and grade > -1:
    print(name + ", your letter grade for " + assignment + " is " + grades[4])
else:
    print(grades[5])

File: grades.js
const readline = require("readline")

let grades = ["A", "B", "C", "D", "F", "INVALID GRADE"];

const r1 = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

r1.question("Please enter your name: ", (name) => {
    r1.question("Assignment: ", (assignment) => {
        r1.question("Thank you. Please enter the grade you wish to convert into a letter: ", (gradeStr) =>{
            let grade = parseInt(gradeStr)
            if(grade <= 100 && grade >=90){
                console.log(name + ", your letter grade for " + assignment + " is " + grades[0])
            }
            else if(grade <= 89 && grade >= 80){
                console.log(name + ", your letter grade for " + assignment + " is " + grades[1])
            }
            else if(grade <= 79 && grade >= 70){
                console.log(name + ", your letter grade for " + assignment + " is " + grades[2])
            }
            else if(grade <= 69 && grade >= 60){
                console.log(name + ", your letter grade for " + assignment + " is " + grades[3])
            }
            else if(grade <= 59 && grade >= 0){
                console.log(name + ", your letter grade for " + assignment + " is " + grades[4])
            }
            else{
                console.log(grades[5])
            }
        })
    })
})
